docs/0_prerequisites.md

```diff
@@ -0,0 +1,18 @@
+# 💼 Prerequisites
+
+In order to use ReVanced Manager, certain requirements must be met.
+
+## 🤝 Requirements
+
+- An Android device running Android 8 or higher
+- Any device architecture except ARMv7
+
+## ⬇️ Installation
+
+- Download and install the latest version of [ReVanced Manager](https://github.com/revanced/revanced-manager/releases/latest)
+
```

Comment:
```suggestion
```
This step should be removed from Prerequisites since there's a dedicated Installation page.